backend/internal/system/healthcheck/model/status.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Package model defines the data structures for the health check module.
+package model
+
+// ServerStatus represents an liveliness details of the server.
+type ServerStatus struct {
+	Status        string          `json:"status,omitempty"`
+	ServiceStatus []ServiceStatus `json:"service_status,omitempty"`
+}
+
+// ServiceStatus represents the status of a service in the system.
+type ServiceStatus struct {
+	ServiceName string `json:"service_name,omitempty"`
+	Status      string `json:"status,omitempty"`
+}
+
+// Status enum represents the possible statuses of a service.
+const (
+	StatusUp      = "UP"
```

Comment:
We can define server statuses as type definitions similar to https://github.com/asgardeo/thunder/blob/be8775cd7c8f176cc537f72ad9c1632fdab6e426/backend/internal/flow/constants/constants.go#L23

Reply:
Fixed with d358ed4e009cbb2bb599ff86f51720a214af6e19